README.md

```diff
@@ -223,7 +223,7 @@ sublime-phpcs is insanely powerful, but if you'd prefer automatic linting, [Subl
 
 ### Visual Studio
 
-Please see "[Setting up PHP CodeSniffer in Visual Studio Code](https://tommcfarlin.com/php-codesniffer-in-visual-studio-code/)", a tutorial by Tom McFarlin.
+Please see "[Setting up PHP_CodeSniffer in Visual Studio Code](https://tommcfarlin.com/php-codesniffer-in-visual-studio-code/)", a tutorial by Tom McFarlin.
```

Comment:
As the title of @tommcfarlin's post, this one shouldn't change (unless Tom is happy to change references in his post).

Reply:
Reverted this one.